test/__renders__/1.html

```diff
@@ -0,0 +1 @@
+<svg>
```

Comment:
There's a small problem with the `OnchainMetadataTest:testRender` now because `forge` in watch mode keeps re-running tests in an infinite loop. I think the solution could be changing the JS to only override file when something changes

Reply:
I don't see where it's running in a loop? Is this still relevant?